core/src/components/textarea/textarea.ionic.outline.scss

```diff
@@ -0,0 +1,72 @@
+@use "../../themes/ionic/ionic.globals.scss" as globals;
+@import "./textarea.vars";
+
+// Textarea Fill: Outline
+// ----------------------------------------------------------------
+
+:host(.textarea-fill-outline) {
+  --border-width: #{globals.$ionic-border-size-025};
+  --border-color: #{globals.$ionic-color-neutral-500};
+}
+
+// Textarea Fill: Outline, Textarea Wrapper
+// ----------------------------------------------------------------
+
+:host(.textarea-fill-outline) .textarea-wrapper-inner {
+  /**
+   * The border should be relative to the inner wrapper
+   * so that it does not include the label.
+   */
+  position: relative;
+}
+
+// Textarea Fill: Outline, Outline Container
+// ----------------------------------------------------------------
+
+:host(.textarea-fill-outline) .textarea-outline {
+  @include position(0, 0, 0, 0);
+  @include border-radius(var(--border-radius));
+
+  position: absolute;
+
+  width: 100%;
+  height: 100%;
+
+  pointer-events: none;
+
+  border: var(--border-width) var(--border-style) var(--border-color);
+}
+
+// Textarea Fill: Outline, Bottom Content
+// ----------------------------------------------------------------
+
+/**
+ * The bottom content should never have
+ * a border with the outline style.
+ */
+:host(.textarea-fill-outline) .textarea-bottom {
+  border-top: none;
+}
+
+// Textarea Fill: Outline, Native Textarea
+// ----------------------------------------------------------------
+
+:host(.textarea-fill-outline) textarea {
+  margin-top: 4px;
+}
+
+// Textarea Fill: Outline, Start / End Slots
+// ----------------------------------------------------------------
+
+:host(.textarea-fill-outline) .start-slot-wrapper,
+:host(.textarea-fill-outline) .end-slot-wrapper {
+  margin-top: 3px;
```

Comment:
Can we use the DS values here, maybe the border token `ionic-border-size-075`? Or maybe we should talk with UX to have outline tokens if we think its not correct to use the ones from border.

Reply:
In my opinion, it can be confusing to use a token that isn't related.

Reply:
I spoke with Beth about this to determine the best token and realized I wasn’t setting the `ion-icon` to the correct scale. Adjusting it to the right size allowed me to remove the start/end margin, and then only `margin-top` of `2px` was needed, which I added using the `$ionic-space-050` token. This has been resolved in [this commit](https://github.com/ionic-team/ionic-framework/pull/29776/commits/e6d53fbb6a3e572082b1999f79f8eb696b14e6cd).